content/implementations/jvm.md

```diff
@@ -189,6 +189,8 @@ java cucumber.api.cli.Main --version
 The [JUnit Runner](#junit-runner) and [Android Runner](#android-runner) can also pick
 up configuration options defined via the `@CucumberOptions` annotation.
 
+**Note:** Cucumber only supports @ClassRule},@BeforeClass and @AfterClass JUnit annotations.
```

Comment:
Is the { significant here?

Reply:
Nope, copy-paste mistake. Will remove it, thanks!